_episodes/08-CI.md

```diff
@@ -475,73 +413,57 @@ new test.
 Codecov is free for open-source projects and integrates very nicely with GitHub.
 
 On your browser, navigate again to the [GitHub
-Marketplace](https://github.com/marketplace) and this time, select the category
-`Code quality`. Find and click on `Codecov`, and on the following page install
-the `Open Source` plan. You can choose to add CodeCov to your accout here, and
-you can choose either to add it to all repositories or to slect repositories.
-After going through the install process, you will be redirected to codecov.io
-where you should log in with your GitHub Accout.
+Marketplace](https://github.com/marketplace) and this time, select the category `Code quality`.
+Find and click on `Codecov`, and on the following page install the `Open Source` plan.
+You can choose to add CodeCov to your account here, and you can choose either to add it to all repositories or to select repositories.
+After going through the installation process, you will be redirected to codecov.io where you should log in with your GitHub Account.
 
-Once on the `Codecov` web page, notice that `molecool` is already in your
-repositories and it already has results to show you. How is this possible?
+Once on the `Codecov` web page, notice that `molecool` is already in your repositories and it already has results to show you. How is this possible?
 
-The last line step of our `CI.yml` configuration file is a call to an executable
-called `codecov`. The CookieCutter includes this utility program and sets it up
-automatically for us. As a result, when our CI workflow runs successfully (all
-tests pass), GitHub Actions runs `codecov`, which send statistics of our test
-coverage automatically to Codecov.io. Neat!
+The last line step of our `CI.yml` configuration file is a call to an executable called `codecov`.
+The CookieCutter includes this utility program and sets it up automatically for us.
+As a result, when our CI workflow runs successfully (all tests pass), GitHub Actions runs `codecov`, which send statistics of our test coverage automatically to Codecov.io. Neat!
 
-Back to the Codecov `molecool` report page, the sunburst plot gives a
-hierachical view of our project and its coverage. 
+Back to the Codecov `molecool` report page, the sunburst plot gives a hierarchical view of our project and its coverage. 
 
 <br />
 <center><img src="../fig/08_CI/codecov_screenshot.png"></center>
 <br />
 
-Red areas represent code that is poorly covered by our tests. On the right, you
-see the latest commits and can click on each one of them to see how they changed
-overall coverage of the project. It is possible, although outside the scope of
-this episode, to setup Travis to fail if a commit lowers the coverage of our
-codebase. The bottom of the page shows a simple file-browser like interface,
-much like the HTML page we obtained before.
+[//]: # TODO: Check if the following feature work in GitHub Action, this paragraph is the legacy from Travis CI
```

Comment:
Good TODO. I don't think I've done this yet. Please add this to our shared doc to address in the next meeting!

Reply:
Sure!